client/src/components/DraggableRow.tsx

```diff
@@ -47,11 +47,21 @@ const DraggableRow = ({
     }
   })
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const [{ isDragging: dragActive }, drag, preview] = useDrag({
+  const [{ isDragging: dragActive }, drag] = useDrag({
     type: itemType,
     item: { uuid: row.uuid, index },
     end: (item, monitor) => {
-      if (!monitor.didDrop()) {
+      if (!item) {
```

Comment:
Not sure I like this solution, it means I have to drop the row exactly on top of another row.